lighthouse-core/audits/load-fast-enough-for-pwa.js

```diff
@@ -64,7 +70,7 @@ class LoadFastEnough4Pwa extends Audit {
     /** @type {string|undefined} */
     let explanation;
     if (!score) {
-      displayValue = [`Interactive at %d\xa0s`, tti.timing / 1000];
+      displayValue = [override?displayValueTextWithOverride: displayValueText, tti.timing / 1000];
```

Comment:
nit: spacing around `?`

Reply:
maybe just set it to a variable above this rather than nesting it in the array